src/proto/robot_status_msg.proto

```diff
@@ -104,6 +105,20 @@ enum MotorFault
     PHASE_W_SHORT_COUNTER_DETECTED = 12;
     PHASE_W_SHORT_TO_GND_DETECTED  = 13;
     PHASE_W_SHORT_TO_VS_DETECTED   = 14;
+
+    /*********** STSPIN Faults ************/
+    NO_FAULT     = 15;
+    DURATION     = 16;
+    OVER_VOLT    = 17;
+    UNDER_VOLT   = 18;
+    OVER_TEMP    = 19;
+    START_UP     = 20;
+    SPEED_FDBK   = 21;
+    OVER_CURR    = 22;
+    SW_ERROR     = 23;
+    SAMPLE_FAULT = 24;
+    OVERCURR_SW  = 25;
+    DP_FAULT     = 26;
```

Comment:
since we have control over the faults now, I'd suggest naming these to be more explicative of the fault